src/workflows/modals/PermissionsModal.tsx

```diff
@@ -238,11 +238,11 @@ export const PermissionsModal = (props: WorkflowPermissionsModalProps) => {
   });
 
   const modalTitle =
-    versionOrNamespace === 'Version' ? 'Edit version permissions' : `Edit permissions for namespace ${namespace}`;
+    versionOrCollection === 'Version' ? 'Edit version permissions' : `Edit permissions for collection ${namespace}`;
```

Comment:
I understand the goal is to change only what is user facing, but I am afraid that the next dev in this code base might be confused by what a namespace is if it is not consistent